src/containers/shared/components/Transaction/XChainBridge.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Account from '../Account'
+import { Amount } from '../Amount'
+import { SimpleRow } from './SimpleRow'
+
+interface XChainBridgeProps {
+  lockingDoor: string
+  lockingIssue: string
+  issuingDoor: string
+  issuingIssue: string
+  signatureReward: string | undefined
+  bridgeOwner: string
+}
+
+export const XChainBridge = (props: XChainBridgeProps) => {
```

Comment:
Why is all of this in a separate component?

Reply:
It gets used in most of the new sidechain txs - I had a bunch of repeated code and kept having to modify it in each branch when I made a change, this was easier